fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkReader.java

```diff
@@ -142,17 +140,20 @@ public Object readItem() throws Exception {
             } else {
                 logger.finer("readItem: Got CosClient successfully!");
             }
-            imported = BulkDataUtils.readFhirResourceFromObjectStore(cosClient, cosBucketName, importPartitionWorkitem,
-                    numOfLinesToSkip, loadedFhirResources, Constants.IMPORT_IS_REUSE_INPUTSTREAM, chunkData);
+            numOfParseFailures = BulkDataUtils.readFhirResourceFromObjectStore(cosClient, cosBucketName, importPartitionWorkitem,
+                    numOfLinesToSkip, loadedFhirResources, chunkData);
             break;
         default:
             break;
```

Comment:
Thinking about this... it's probably worth adding a logger.finer() just in case the code changes and hits default

Reply:
make sense. adding  logger.warning("readItem: Data source storage type not found!");